extensions/wikia/PremiumPageHeader/scripts/wiki-header.js

```diff
@@ -0,0 +1,41 @@
+require(['wikia.window', 'jquery', 'wikia.tracker'], function (window, $, tracker) {
+	'use strict';
+
+	$(function () {
+		var track = tracker.buildTrackingFunction({
+				category: 'premium-page-header',
+				trackingMethod: 'analytics'
+			}),
+			$wikiHeader = $('.pph-wiki-header');
+
+		$wikiHeader.find('.wordmark').on('click', function () {
+			track({
+				action: tracker.ACTIONS.CLICK,
+				label: 'wordmark-image'
+			});
+		});
+
+		$wikiHeader.find('.pph-wordmark-text').on('click', function () {
+			track({
+				action: tracker.ACTIONS.CLICK,
+				label: 'wordmark-text'
+			});
+		});
+
+		$wikiHeader.find('.pph-tally').on('click', function () {
```

Comment:
does it make any sense to track tally if there is no link and even a cursor does not change to pointer on hover? absolutely nothing indicates that this is a clickable element, so I would say that results of tracking will give us no useful information